ReactNativeClient/lib/layout-utils.js

```diff
@@ -1,9 +0,0 @@
-const layoutUtils = {};
-
-layoutUtils.size = function(preferred, min, max) {
-	if (preferred < min) return min;
-	if (typeof max !== 'undefined' && preferred > max) return max;
-	return preferred;
-};
-
-module.exports = layoutUtils;
```

Comment:
I believe this file isn't used anywhere in the codebase, so I deleted it. Lmk if I missed something with my global search though!